src/crimson/os/seastore/lba_manager.cc

```diff
@@ -22,7 +22,7 @@ LBAManager::update_mappings(
       extent->get_paddr(),
       nullptr	// all the extents should have already been
 		// added to the fixed_kv_btree
-    );
+    ).discard_result();
```

Comment:
seems needn't discard here. update_mappings_ret is base_iertr::future<>;  update_mapping also return base_iertr::future<>, no return value need be discarded.